app/models/concerns/imageable.rb

```diff
@@ -6,6 +6,10 @@ module Imageable
 
   included do
     has_one_attached :image, dependent: :destroy
+    validates :image, content_type: { in: ["image/png", "image/jpg", "image/jpeg", "image/gif"],
+                                      message: I18n.t("manifold.error.content_type_invalid_image") },
+                      size: { less_than: 700.kilobyte ,
+                              message: I18n.t("manifold.error.file_size_out_of_range_image") }
```

Comment:
- Tests if size exceeds 700kB
- Hard coded numbers (700) should be specified in `config`

Reply:
Added test and numeric sizes to config.